### PR TITLE
Add Orca Pytorch Metrics (Probabilistic Metrics)

### PR DESCRIPTION
**Pull request by GZHoffie**
I add some classification metrics, precision and recall, in this PR. I am not very sure of the input format of predictions and targets, so more tests are needed. I may also add more metrics later, such as F1 Score.

### CODE REVIEW

**Comment by GZHoffie** [comment]
Added other probabilistic metrics:

- [ ] Binary Cross Entropy
- [ ] Categorical Cross Entropy
- [ ] Sparse Categorical Cross Entropy
- [ ] Kullback-Liebler Divergence
- [ ] Poisson

File: pyzoo/zoo/orca/learn/pytorch/pytorch_metrics.py [hunk 1]
```diff
@@ -191,6 +191,34 @@ def __call__(self, preds, targets):
     def compute(self):
         return self.correct.float() / self.total
 
+class MSE:
```
Comment (yangw1234) [inline pyzoo/zoo/orca/learn/pytorch/pytorch_metrics.py:194]: duplicated?

**Comment by yangw1234** [comment]
LGTM, we can merge it after jenkins pass.

File: pyzoo/zoo/orca/learn/pytorch/pytorch_metrics.py (hunk 1 above)
Comment (GZHoffie) [reply pyzoo/zoo/orca/learn/pytorch/pytorch_metrics.py:195]: It is marked as added after the merge. The original `class MSE` on `master` is marked as deleted...

**Comment by GZHoffie** [comment]
Jenkins passed: http://10.239.47.210:18888/view/ZOO-PR-PIPELINE/job/ZOO-PR-Validation/5171/, the case that failed is run again here: http://10.239.47.210:18888/job/ZOO-PR-Validation-Spark-2.4/3538/

**Pull request merged**